.github/labeler.yml

```diff
@@ -0,0 +1,18 @@
+sig-api:
+  - operators/**/*
+
+sig-ui:
+  - webservice/**/*
+
+sig-operations:
+  - infrastructure/**/*
+  - provisioning/**/*
```

Comment:
Should we also add the user-environment label to the VM provisioning part?

Reply:
Right!